### DISCUSSION TITLE
Advice on implementing MLFLOW in Kedro

### DISCUSSION THREAD

**Discussion by Jason-F** [kedro-org/kedro#1524] [Idea]
Does anyone know of a good way to implement mlflow in kendro? Maybe an example project? just wanna output the results to a csv file but unsure how to integrate mlflow to an existing model

**Comment by noklam** [comment]
There is a `kedro-mflow` plugin built by our community.
https://github.com/Galileo-Galilei/kedro-mlflow

**Reply by Jason-F** [reply]
Thank you for that link. I'll dive into it!

**Reply by Jason-F** [reply]
> 
@[noklam](https://github.com/noklam) I can't use kedro-mflow due to conflicts occuring. Any way to easily implement mlflow?

**Reply by datajoely** [reply]
What conflicts are you getting?

**Reply by Jason-F** [reply]
> What conflicts are you getting?

Different kedro version than one suggested and fears of conflicts occuring during expansion of the code base/dependencies

**Reply by Galileo-Galilei** [reply]
Hi @Jason-F Just for the record, you are probably unlucky: the recent ``kedro==0.18.1`` (3 days ago) release broke the plugin. 

You can either: 
-  use ``kedro<=0.18.0`` for now, everything will work fine.
- Wait for this weekend,  I'll commit a fix for 0.18.1.
